specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/examples/managedApisGet.json

```diff
@@ -0,0 +1,65 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "location": "northcentralus",
+    "apiName": "testManagedApi",
+    "api-version": "2016-06-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "name": "testManagedApi",
+          "connectionParameters": {
+            "token": {
+              "type": "oauthSetting",
+              "oAuthSettings": {
+                "identityProvider": "oauth2",
+                "clientId": "totallyrealclientid",
```

Comment:
might need some more serious string for official documentation

Reply:
Changed